typescript/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "outDir": "./dist",
```

Comment:
Could you move the index.html in the dist folder as well? Currently the dist folder contains the app.js file only. We would like to be able to just take the dist and deploy it as is.

Thank you.